_packages/native-preview/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "module": "node16",
```

Comment:
Not `nodenext`?

Reply:
I just copied it from another dir, I can do `nodenext` of course